build_system/config_android.py

```diff
@@ -21,13 +21,14 @@
 
 android_config.set_file_abis({
     c.arch_arm: "armeabi-v7a",
-    c.arch_x86: "x86"
+    c.arch_x86: "x86",
+    c.arch_arm64: "arm64-v8a"
 })
 
 #-----------------------------------------------------------------------
 def build_node_js(config):
     return [
-        """android-gcc-toolchain $ARCH --api 17 --host gcc-lpthread -C \
+        """android-gcc-toolchain $ARCH --api 21 --host gcc-lpthread -C \
```

Comment:
Only need to set api to 21 when build arm64, maybe return different config for different abi.